tests/xls/test_xls_processing.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from uuid import uuid4
+
+import pytest
+
+from megaparse.parser.llama import LlamaParser
+from megaparse.parser.megaparse_vision import MegaParseVision
+from megaparse.main import MegaParse
+
+from langchain_core.language_models import FakeListChatModel
+
+
+@pytest.mark.asyncio
+async def test_megaparse_xls_processor():
+    p = Path("./tests/xls/file_example_XLS_50.xls")
+    processor = MegaParse()
+    result = await processor.aload(file_path=p)
+    assert len(result) > 0
+
+
+@pytest.mark.asyncio
+async def test_megaparse_xlsx_processor():
+    p = Path("./tests/xls/file_example_XLSX_50.xlsx")
+    processor = MegaParse()
+    result = await processor.aload(file_path=p)
+    assert len(result) > 0
```

Comment:
We should have a longer list of assertion here to verify parsing has worked the way we want

Reply:
Same created a test ticket #core-266